### PR TITLE
Style Catalog Page and Add Class Selector

### PR DESCRIPTION
**Pull request by nijat-ahmadli**
- Adjust Catalog Css to make it mobile friendly

[Screenshot - Pod](https://user-images.githubusercontent.com/8284418/76743620-d078e680-676a-11ea-8da9-dea4eb6edd9a.png)
[Screenshot - Select class type](https://user-images.githubusercontent.com/8284418/76743625-d2db4080-676a-11ea-9f3f-3b6d53251092.png)

### CODE REVIEW

File: apps/sse_cfg/priv/docroot/catalog/catalog.css [hunk 1]
```diff
@@ -1,6 +1,19 @@
 @charset "utf-8";
 @import "/common/sse.css";
 
+:root {
```
Comment (jacobythwaites) [inline apps/sse_cfg/priv/docroot/catalog/catalog.css]: oooo fancy

File: apps/sse_cfg/priv/docroot/catalog/catalog.css [hunk 2]
```diff
@@ -13,26 +26,26 @@ publisher {
 }
 
 publisher > header {
-  font-size: medium;
-  background: #ebebeb;
-  color: #647276;
-  border-bottom: 1px solid #eee;
+  color: var(--onfido-blue, blue);
```
Comment (jacobythwaites) [inline apps/sse_cfg/priv/docroot/catalog/catalog.css]: Can't see the point of the fallback

File: apps/sse_cfg/priv/docroot/catalog/catalog.css (hunk 1 above)
Comment (nijat-ahmadli) [reply apps/sse_cfg/priv/docroot/catalog/catalog.css]: We might want to add this to `common/sse.css` later to be able to reuse in other pages.

File: apps/sse_cfg/priv/docroot/catalog/catalog.css [hunk 3]
```diff
@@ -13,26 +26,26 @@ publisher {
...
-  padding: 5px 5px;
-  background-color: #fafafa;
+  padding: 0 5px 3px 0;
+  width: 100%;
 }
+
+entry div {
+  font-size: 0.8em;
+  color: gray;
+}
```
[the remark this answers is not in the document]
Comment (nijat-ahmadli) [reply apps/sse_cfg/priv/docroot/catalog/catalog.css]: Is it just a convention?

[the remark this answers is not in the document]
Comment (lucafavatellawork) [reply apps/sse_cfg/priv/docroot/catalog/catalog.css]: I opened https://github.com/opensparkl/sse_core/pull/129 so to ease respecting this if we want to.

**Comment by nijat-ahmadli** [comment]
@lucafavatellawork I usually prefer to rebase instead of merge to avoid redundant merge commits when syncing my branch. Thanks for doing it for me though anyways :)

File: apps/sse_cfg/priv/docroot/catalog/catalog.css (hunk 3 above)
[the remark this answers is not in the document]
Comment (jacobythwaites) [reply apps/sse_cfg/priv/docroot/catalog/catalog.css]: Yes, convention. But please let's keep it.

File: apps/sse_cfg/priv/docroot/catalog/catalog.css (hunk 1 above)
Comment (jacobythwaites) [reply apps/sse_cfg/priv/docroot/catalog/catalog.css]: Yes, nice idea. Don't worry for now though.

Comment (nijat-ahmadli) [reply apps/sse_cfg/priv/docroot/catalog/catalog.css]: I thought I added new line in my latest changes, but looks like going to the next line doesn't add a new line in SublimeText. Add another one :)

**Pull request closed without merging**